ur5_moveit_config/launch/moveit_rviz.launch

```diff
@@ -11,6 +12,7 @@
   <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false"
 	args="$(arg command_args)" output="screen">
     <rosparam command="load" file="$(find ur5_moveit_config)/config/kinematics.yaml"/>
+    <param name="arm_prefix" value="$(arg arm_prefix)"/>
```

Comment:
This is required because the URKinematics plugin is dynamically linked to the RViz process. The URKinematics plugin looks for the private parameter arm_prefix, which has to be set inside the RViz namespace.